internal/terraform/terraform.go

```diff
@@ -184,6 +186,12 @@ func DestroyBaseTerraform(skipBaseTerraform bool) {
 			envs["TF_VAR_capacity_type"] = "SPOT"
 		}
 
+		err = aws.DestroyLoadBalancer(viper.GetString("cluster-name"))
+		if err != nil {
```

Comment:
not sure what we want here, double checking... it will break/stop the destroy command if it's not possible to delete the load balancer, is that what we want?

Reply:
I think so, because the destroy will fail anyway if load balancer is not successfully removed, the vpc has dependencies with the load balancer and the terraform breaks.